pages/learn/deploy/offline-updates.md

```diff
@@ -0,0 +1,273 @@
+---
+title: Offline updates
+excerpt: An efficient approach for updating previously deployed IoT devices when they’re offline by design.
+---
+
+# Offline updates
+
+Offline updates is an efficient process to update devices without needing an internet connection. It involves preloading a image and reflashing it to on a device in a way that it retains its unique balena identity, apps, and configurations. This is helpful in scenarios where device's internet access is unavailable, limited, blocked or even when it's air-gapped. 
+
+## Overview of the process
+
+When a device is reflashed, it defaults back to a factory state. The device gets a new identity, a new API key, and updated `config.json` settings. All services, data and logs stored on the device is erased permanently. 
+
+With the offline upgrades process, the device still resets to a factory default state, all the preserving its identity in `config.json` with a new API key and then using `balena preload` to load an updated application release. That way, the device will pick up right where it left off with the same name and UUID but with an updated application release or/and balenaOS upgrade. Broad steps in the process include:
```

Comment:
> all the preserving its identity in `config.json` with a new API key

I don't get that. Do you mean v?

```suggestion
With the offline upgrades process, the device still resets to a factory default state and a new API key, while preserving its identity in `config.json` and then using `balena preload` to load an updated application release. That way, the device will pick up right where it left off with the same name and UUID but with an updated application release or/and balenaOS upgrade. Broad steps in the process include:
```

Reply:
Oh yeah, that doesn't look good. Here's the improved snippet. 

```
With the offline updates process, the device still resets to a factory default state and adds a new API key, while preserving its identity in `config.json`. It uses [`balena preload`][balena-preload] to load an updated application release. That way, the device will pick up right where it left off with the same name and UUID but with an updated application release or/and balenaOS upgrade. Broad steps in the process include:
```